app/controllers/booking_controller.rb

```diff
@@ -1,2 +1,28 @@
 class BookingController < ApplicationController
+  def index
+
+  def accept
+    @booking = Booking.find(params[:id])
+    @booking.status = "accepted"
+    @booking.save
+    redirect_to bookings_path
+  end
+
+  def reject
+    @booking = Booking.find(params[:id])
+    @booking.status = "rejected"
+    @booking.save
+    redirect_to bookings_path
+  end
+end
+
+private
+
+def set_venue
+  @venue = Venue.find(params[:id])
+end
+
+def venue_params
+  params.require(:venue).permit(:name, :date)
```

Comment:
![image](https://github.com/Sols777/project-week/assets/84518797/88dd553d-e26d-45fe-b3b7-3325e09d8321)
 shouldnt the params also contain the rest of the fields like address ,description , capacity and bustability.